local/notebooks/analysis.py

```diff
@@ -11,21 +11,35 @@
 def run_state_dict(datasets, label, mask, save_fig):
     """Runs an analysis across state for multiple dictionaries
```

Comment:
'an analysis' (here and in other functions) isn't very specific. What is the analysis?